source/reference/read-concern-majority.txt

```diff
@@ -10,6 +10,8 @@ Read Concern ``"majority"``
    :depth: 1
    :class: singlecol
 
+.. _readconcern-majority:
```

Comment:
You don't need to add these refs. `.. readconcern:: "majority"` you can already point to this place in the docs by using:

```
:readconcern:`"majority"`
```